multisrc/overrides/madara/randomscan/src/luratoon.kt

```diff
@@ -1,4 +1,4 @@
-package eu.kanade.tachiyomi.extension.pt.randomscan
+package eu.kanade.tachiyomi.extension.pt.luratoon
```

Comment:
Preserve the old package name so users get updates through the app.

Reply:
done

Reply:
done